modules/oracle-xe/src/main/java/org/testcontainers/containers/OracleContainer.java

```diff
@@ -0,0 +1,73 @@
+package org.testcontainers.containers;
+
+import org.testcontainers.utility.TestcontainersConfiguration;
+
+/**
+ * @author gusohal
+ */
+public class OracleContainer extends JdbcDatabaseContainer {
+
+    public static final String NAME = "oracle";
+    public static final String IMAGE = TestcontainersConfiguration.getInstance()
+            .getProperties().getProperty("oracle.container.image","wnameless/oracle-xe-11g");
+    private static final int ORACLE_PORT = 1521;
+    private static final int APEX_HTTP_PORT = 8080;
+
+    public OracleContainer() {
+        super(IMAGE + "@sha256:15ff9ef50b4f90613c9780b589c57d98a8a9d496decec854316d96396ec5c085");
```

Comment:
Since the Oracle image we use is owned by a third party and thus hard to reliably pin, I'd like to use the sha256 hash to lock down the version. Changes in `RemoteDockerImage` permit this to work.

Why lock down the version? As we're importing this module into the main respository our stability, and ability to release, rely on all the modules builds staying repeatable. I'd like to do this with other dependencies too, but this one in particular is a crucial one.